pkg/middlewares/metrics/metrics.go

```diff
@@ -131,6 +144,88 @@ func getMethod(r *http.Request) string {
 	return r.Method
 }
 
+func getRequestTLSVersion(req *http.Request) string {
+	var tlsVersion string
+
+	switch req.TLS.Version {
+	case tls.VersionTLS10:
+		tlsVersion = "tls-1.0"
+	case tls.VersionTLS11:
+		tlsVersion = "tls-1.1"
+	case tls.VersionTLS12:
+		tlsVersion = "tls-1.2"
+	case tls.VersionTLS13:
+		tlsVersion = "tls-1.3"
+	case tls.VersionSSL30: //nolint
```

Comment:
why nolint here?

Reply:
Because VersionSSL30 will be removed in Go 1.14 so the linter throws an error about that.

Reply:
Shouldn't SSL (not TLS) be dead dead?